js/languages/en.js

```diff
@@ -249,7 +250,6 @@ export default {
     doneify: 'Mark as done',
     ds: 'S  M  T  W  Θ  F  S',
     'enable-lists': 'Enable club/staff lists',
-    events: 'Events',
```

Comment:
For some dumb reason, the Events header has always been inserted by JavaScript?? It has now been moved to the HTML, as reflected in the localization text